### ISSUE TITLE
tex2pdf2word

### ISSUE THREAD

**Issue by liuluping0106** [dothinking/pdf2docx#87]
Pdf2docx can not convert `\begin{cases}` and `\frac{}{}` correctly.

**Comment by dothinking** [comment]
Do you mean math equation/formula/expression? If so, you're correct -> these are not typical text, `pdf2docx` doesn't parse equation for now. But would be a good idea to include this feature.

**Comment by liuluping0106** [comment]
Yes, I generate pdf from tex. When I convert this pdf to word, math equation can not be translate correctly. I think if analysis math equation is difficult, we can just locate all equation and translate them to picture. Hope this make sense!

**Comment by dothinking** [comment]
Agree, that's also what in my mind. Hopefully it can be enhanced in two stages:

- in short term, recognize the bbox of equation and crop a picture accordingly as suggested -> should not so hard
- in long term, recreate the equation from image -> there's machine learning method on this topic, but need more time to explore this
